Fix (internationalization): Simplified Chinese fills in gaps and removes extra content, optimizes description

Fortytwoo opened the pull request:
Fill in the missing Chinese Simplified Translation

It's now 100 percent complete.

<img width="1470" height="857" alt="image" src="https://github.com/user-attachments/assets/76a9f668-f554-4544-b531-d3fb92a0fd46" />


cc:
- @potiuk
- @Lee-W 
- @gyli

gyli commented on airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/common.json:
```diff
@@ -93,6 +100,12 @@
     "or": "或"
   },
   "filter": "筛选",
+  "filters": {
```
As suggested here https://github.com/apache/airflow/pull/55471#discussion_r2338324515
I propose following translation:
```
  "filters": {
    "logicalDateFrom": "逻辑日期起始",
    "logicalDateTo": "逻辑日期结束",
    "runAfterFrom": "执行时间起始",
    "runAfterTo": "执行时间结束"
  },
```

gyli commented on airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/hitl.json:
```diff
@@ -23,6 +25,7 @@
     "approvalRequired": "需要批准",
     "choiceReceived": "已选择",
     "choiceRequired": "需要选择",
+    "noResponseReceived": "未收到响应",
```
It could be confusing that both `response` and `action` above are translated into `响应`。

I suggest translating `requiredAction` into `需要操作的`。

If you agree, the one in `airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/dashboard.json` also needs to be updated.

Lee-W replied:
i like what's suggested

gyli commented on airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/hitl.json:
```diff
@@ -23,6 +25,7 @@
     "approvalRequired": "需要批准",
     "choiceReceived": "已选择",
     "choiceRequired": "需要选择",
+    "noResponseReceived": "需要操作的",
```
```suggestion
    "noResponseReceived": "未收到响应",
```

I am not sure which is better, `响应` or `回应`

Fortytwoo replied:
use **响应**?
 I think it should be consistent with `"responseRequired": "需要响应"`

gyli replied:
It's not about this specific line, but about the translation of "response" in general.
I checked the AWS and Azure official doc about their service API, and seems like they both use `响应` for "response", so I think we are good here.

gyli commented on airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/dashboard.json:
```diff
@@ -33,7 +33,7 @@
+    "activeDags": "已启用的 Dags",
     "failedDags": "失败的 Dags",
     "queuedDags": "排队的 Dags",
     "requiredActions": "待响应的任务实例",
```
```suggestion
    "requiredActions": "待操作的任务实例",
```

gyli commented on airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/hitl.json:
```diff
@@ -8,6 +8,8 @@
   },
   "requiredAction_one": "待响应的任务实例",
```
```suggestion
  "requiredAction_one": "待操作的任务实例",
```

`requiredAction` -> `待操作的`
and all others below, you get the idea

gyli commented on airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/hitl.json:
```diff
@@ -6,8 +6,10 @@
   "requiredAction_other": "待响应的任务实例",
+  "requiredActionCount_one": "需要响应的任务实例 ({{count}})",
+  "requiredActionCount_other": "需要响应的任务实例 ({{count}})",
   "requiredActionState": "待响应的任务实例状态",
```
```suggestion
"requiredAction_other": "待操作的任务实例",
  "requiredActionCount_one": "待操作的任务实例 ({{count}})",
  "requiredActionCount_other": "待操作的任务实例 ({{count}})",
"requiredActionState": "待操作的任务实例状态",
```

Fortytwoo committed:
feat(i18n): Add Chinese (zh-CN) internationalization support

This commit adds Simplified Chinese (zh-CN) internationalization support to Airflow UI.

The changes include:
- Added complete Simplified Chinese translation files
- Updated i18n configuration to include zh-CN locale
- Updated translation completeness check script
- Updated CODEOWNERS for zh-CN translations
- Optimized translations for better user experience
- Unified terminology for consistency (e.g. backfill, login)
- Updated DAG/DAGs capitalization according to #55099

Co-authored-by: @gyli

Fortytwoo committed:
Merge branch 'main' into feature/i18n-chinese

kaxil commented on airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/hitl.json:
```diff
@@ -6,9 +6,11 @@
+"requiredAction_other": "待操作的任务实例",
+  "requiredActionCount_one": "待操作的任务实例 ({{count}})",
+  "requiredActionCount_other": "待操作的任务实例 ({{count}})",
+"requiredActionState": "待操作的任务实例状态",
```
```suggestion
  "requiredAction_other": "待操作的任务实例",
  "requiredActionCount_one": "待操作的任务实例 ({{count}})",
  "requiredActionCount_other": "待操作的任务实例 ({{count}})",
  "requiredActionState": "待操作的任务实例状态",
```

kaxil committed:
Apply suggestions from code review

kaxil committed:
fixup! Apply suggestions from code review
